### DISCUSSION TITLE
Admin List Search generates Prisma Error

### DISCUSSION THREAD

**Discussion by kidd-dev** [keystonejs/keystone#8986] [Questions]
In the admin area, when I try to search on a list, I get the following error: "Prisma error: Inconsistent column data: Error creating UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-zA-Z], found `u` at 1"

I don't understand why a search would be trying to create a UUID. 

I'm running keystone 4.2.1, keystone-6/auth 7.0.0, keystone-6/core 5.0.0, keystone-6/fields-document 7.0.0, prisma (and client/generator) 4.16.2. Node version is 20.10.0.  

This happens on all lists, regardless of searchFields configuration. I'm fairly new with keystone, but have poked around the source code. I can't figure out what could be affecting this is. I can share keystone.ts or schema as needed.

If anyone can shed any light on where I should go to keep debugging this, I'd be appreciative.

**Comment by vjerryv** [comment]
Same here

**Reply by kennedybaird** [reply]
@vjerryv - what version of keystone are you using? v5 has been [feature frozen / maintenance mode](https://github.com/keystonejs/keystone-5/issues/21) for a few years.

**Reply by vjerryv** [reply]
Keystone6/core 6.1.0 , was using 5.1.0, the error shown after upgraded to 6.1.0

**Reply by kennedybaird** [reply]
If you could provide a very minimal reproduction for me to check, that would be appreciated :+1:

**Reply by vjerryv** [reply]
![image](https://github.com/user-attachments/assets/4c0b1bba-b16f-436d-824b-eaf3084dc45c)
when I search things from the search bar, the error will be occurred

**Reply by vjerryv** [reply]
"dependencies": {
    "@graphql-tools/schema": "^10.0.4",
    "@keystone-6/auth": "^8.0.0",
    "@keystone-6/core": "^6.1.0",
    "@keystone-6/fields-document": "^9.0.0",
    "@keystone-ui/fields": "^7.2.0",
    "dotenv": "^16.0.3",
    "firebase-admin": "^11.5.0",
    "helmet": "^6.0.0",
    "jimp": "^0.16.2",
    "nodemailer": "^6.9.1",
    "typescript": "^4.8.0",
    "uuid": "^10.0.0",
    "yup": "^0.32.11"
  },
  "devDependencies": {
    "@types/jimp": "^0.2.28",
    "@types/uuid": "^10.0.0"
  }

**Reply by kennedybaird** [reply]
Thanks, can you give me your `District` list? Using a couple of basic Task / People lists I can't replicate - using the exact same deps
